mappings/net/minecraft/world/gen/DensityFunctions.mapping

```diff
@@ -0,0 +1,445 @@
+		ARG 0 function
+		ARG 1 value1
+		ARG 2 value2
+	METHOD m_khwsifkt shiftA (Lnet/minecraft/unmapped/C_cjzoxshv;)Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_kjuqbsgg cache2d (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function
+	METHOD m_klpakgay noise (Lnet/minecraft/unmapped/C_cjzoxshv;D)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 noise
+	METHOD m_lwuiythu mappedNoise (Lnet/minecraft/unmapped/C_cjzoxshv;DDD)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 noise
+	METHOD m_mbnvilep weirdScaledSampler (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_cjzoxshv;Lnet/minecraft/unmapped/C_mdkmcezi$C_ufxlyrzf$C_edgplzwm;)Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_mhoauvtb flatCache (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function
+	METHOD m_mifrqffk multiply (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function1
+		ARG 1 function2
+	METHOD m_nhcwyrbz blendAlpha ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_nnktiwfj rangeChoice (Lnet/minecraft/unmapped/C_kzwapyud;DDLnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 input
+		ARG 1 minInclusive
+		ARG 3 maxInclusive
+		ARG 5 whenInRange
+		ARG 6 whenOutOfRange
+	METHOD m_nqecyxgz cacheOnce (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function
+	METHOD m_nsnpgtzo bootstrap (Lnet/minecraft/unmapped/C_tqxyjqsk;)Lcom/mojang/serialization/Codec;
+		ARG 0 registry
+	METHOD m_okrngdwc noise (Lnet/minecraft/unmapped/C_cjzoxshv;DD)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 noise
+	METHOD m_ovaexbma add (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function1
+		ARG 1 function2
+	METHOD m_pfuzzaai constant (D)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 value
+	METHOD m_pvonorqa makeCodec (Lcom/mojang/serialization/MapCodec;)Lcom/mojang/serialization/Codec;
+	METHOD m_qjocdzol noise (Lnet/minecraft/unmapped/C_cjzoxshv;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 noise
+	METHOD m_qwccqgbn mappedNoise (Lnet/minecraft/unmapped/C_cjzoxshv;DDDD)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 noise
+	METHOD m_qwpxedjm map (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_mdkmcezi$C_ucbebgzr$C_vgrinicv;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 input
+		ARG 1 type
+	METHOD m_rrapcnud slide (Lnet/minecraft/unmapped/C_ulvgadxi;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 settings
+		ARG 1 input
+	METHOD m_skodlkvx cacheAllInCell (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function
+	METHOD m_sswjlstj mapFromUnitTo (Lnet/minecraft/unmapped/C_kzwapyud;DD)Lnet/minecraft/unmapped/C_kzwapyud;
```

Comment:
This doesn't seem right, how about
- `mapUnitToValue`
- `mapFromUnitToValue`
- `mapValue`
?

Reply:
The middle one looks good to me :+1: